docs/openapi/components/schemas/common/CertificateOfOrigin.yml

```
@@ -0,0 +1,94 @@
+$linkedData:
+  term: CertificateOfOrigin
+  '@id': https://w3id.org/traceability#CertificateOfOrigin
+title: Certificate Of Origin
+description: Certificate Of Origin
+type: object
+properties:
+  '@context':
+    type: array
+    const:
+      - 'https://www.w3.org/2018/credentials/v1'
+      - 'https://w3id.org/traceability/v1'
+  type:
+    type: array
+    const:
+      - VerifiableCredential
+      - CertificateOfOrigin
+  id:
+    type: string
+  name:
+    type: string
+  description:
+    type: string
+  issuanceDate:
+    type: string
+  issuer:
+    type: object
+  credentialSubject:
+    type: object
+  proof:
+    type: object
+  relatedLink:
+    title: Related Link
+    description: Links related to this verifiable credential
+    type: array
+    items:
+      $ref: ./LinkRole.yml
+  manufacturingCountry: 
+    title: Manufacturing Country
+    description: Manufacturing country.
+    type: string
+    $linkedData:
+      term: countryOfOrigin
+      '@id': https://w3id.org/traceability#countryOfOrigin
+  dateOfExport:
+    title: Date of Export
+    description: The date, time, date time, or other date time value when this supply chain consignment will exit, or has exited from the last port, airport, or border post of the country of export.
```

Turns @mkhraisha's suggestion into a suggestion, and fixes a plurality disagreement.
```suggestion
    description: The date, time, date time, or other date time value when the subject line item(s) will exit, or has(have) exited from the last port, airport, or border post of the country of export.
```